### ISSUE TITLE
[Accessibility] Cannot switch to System Menu from MenuStrip by pressing Left/Right arrow in the first or last MenuItem

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#8326]
### .NET version

8.0.100-alpha.1.22574.1

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No

### Issue description
Related to https://github.com/dotnet/winforms/issues/7770

In **MenuStrip**, when switching through menu items using keyboard (left and right arrow keys), user cannot switch to System menu. It enumerates only it's own items, in a loop. 

For example, menuStrip has 3 MenuItems, Narrator says this: "Item1" -> "Item2" -> "Item3" -> "Item1" -> ...
![image](https://user-images.githubusercontent.com/26474449/205541127-3098f993-981c-46dd-92a8-d04866ded381.png)
![MenuStrip_NoSystemMenu](https://user-images.githubusercontent.com/26474449/205541197-5f13bb46-b821-4e0c-8ac7-156cebeaaa34.gif)

But in .Net framework, there has a **MainMenu** control, use keyboard (left and right arrow keys) user could switch to System menu.

Narrator says this: "Item1" -> "Item2" -> "Item3" -> **"System menu"** -> "Item1" -> ...
![image](https://user-images.githubusercontent.com/26474449/205541627-2d70e6a1-f29f-45c7-979c-5fcc96c9d537.png)
![MainMenu-SystemMenu](https://user-images.githubusercontent.com/26474449/205541674-96a90abc-e28d-4a6e-ba00-83bbafbadc31.gif)

### Steps to reproduce

Here is the sample for MenuStrip .Net 8.0 application:
[MenuStrip.Net8.0.zip](https://github.com/dotnet/winforms/files/10150403/MenuStrip.Net8.0.zip)
[MainMenu.NetFramework.zip](https://github.com/dotnet/winforms/files/10150419/MainMenu.NetFramework.zip)
1. Open above MenuStrip.Net8.0 application
2. Build and Run
3. Open Narrator, press Alt key to focus on the first MenuItem: Item1
4. Press right key several times to observe, after Item3, it will focus on which item

**Comment by merriemcgaw** [comment]
This is a split out issue from an issue submitted by a contributor.

**Comment by merriemcgaw** [comment]
@Tanya-Solyanik do you think this is something we'd be able to do in .NET 9 or would this be more of a .NET 9 thing?

**Comment by Tanya-Solyanik** [comment]
I'd move it to NET9 as the fix might be risky

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang  - Is there a dedicated key stroke to access the system menu?

**Comment by MandiMan** [comment]
@Tanya-Solyanik There is no dedicated key to access the system menu in the core app, but alt+space opens the system menu drop-down list.

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw  - sounds like the user is not blocked, I suggest closing this issue.

**Comment by merriemcgaw** [comment]
I would 100% agree there.
